pkg/sloop/queries/respayloadquery_test.go

```diff
@@ -146,12 +146,39 @@ func Test_GetResPayload_True_PreviousKeyFound(t *testing.T) {
   "payloadKey": "/watch/001546398000/someKind/someNamespace/someName/1546398245000000006",
   "payloadTime": 1546398245000000006,
   "payload": "{\n  \"metadata\": {\n    \"name\": \"someName\",\n    \"namespace\": \"someNamespace\",\n    \"uid\": \"6c2a9795-a282-11e9-ba2f-14187761de09\",\n    \"creationTimestamp\": \"2019-07-09T19:47:45Z\"\n  }\n}"
- },
- {
-  "payloadKey": "/watch/001546394400/someKind/someNamespace/someName/1546398245000000006",
-  "payloadTime": 1546398245000000006,
-  "payload": "{\n  \"metadata\": {\n    \"name\": \"someName\",\n    \"namespace\": \"someNamespace\",\n    \"uid\": \"6c2a9795-a282-11e9-ba2f-14187761de09\",\n    \"creationTimestamp\": \"2019-07-09T19:47:45Z\"\n  }\n}"
  }
 ]`
 	assertex.JsonEqual(t, expectedRes, string(res))
 }
+
+var somePayloadTs = time.Date(2019, 3, 1, 3, 4, 0, 0, time.UTC)
+
+func Test_removeDupePayloads_emptyWorks(t *testing.T) {
+	ret := removeDupePayloads([]PayloadOuput{})
+	assert.Equal(t, []PayloadOuput{}, ret)
+}
+
+func Test_removeDupePayloads_twoUnique_returnsTwo(t *testing.T) {
+	input := []PayloadOuput{
+		{PayLoadTime: somePayloadTs.Add(time.Minute).Unix(), Payload: "abc"},
```

Comment:
s/somePayloadTs.Add(time.Minute).Unix()/somePayloadTs.Add(time.Minute).UnixNano(), payload time is unixnano

Reply:
thanks, fixed